deploy/manifests/README.md

```diff
@@ -22,7 +22,7 @@ To build the manifests, run:
 ```bash
 $ bazel build //deploy/manifests:cert-manager
 # Alternatively, build the openshift variant with:
-$ bazel build //deploy/manifests:cert-manager-openshift
+$ bazel build //deploy/manifests:cert-manager-openshift.yaml
```

Comment:
This change isn't needed, and makes this entry not like the other entries 😅

Reply:
oops this is a documentation change from before i renamed the variable